src/components/state.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, ReactNode, useState } from 'react';
+import { Person } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MyContext = createContext<any>(null);
+
+interface MyProvideProps {
```

Comment:
The name of the interface `MyProvideProps` is not descriptive enough. Consider renaming it to something that better represents its purpose, like `MyProviderProps`.